sw/device/lib/base/testing/BUILD

```diff
@@ -0,0 +1,55 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "testing",
+    srcs = [
+        "mock_mmio.cc",
+        "//sw/device/lib/base:mmio_src",
+    ],
+    hdrs = [
+        "mock_mmio.h",
+        "mock_mmio_test_utils.h",
+    ],
+    defines = [
+        "MOCK_MMIO=1",
+        "MOCK_CSR=1",
+    ],
+    deps = [
+        "//sw/device/lib/base",
+        "@googletest//:gtest",
+    ],
+)
+
+cc_library(
+    name = "mock_mmio",
+    srcs = [
+        "mock_mmio.cc",
+    ],
+    hdrs = [
+        "mock_mmio.h",
+        "mock_mmio_test_utils.h",
+    ],
+    defines = ["MOCK_MMIO=1"],
+    deps = [
+        "//sw/device/lib/base",
+        "@googletest//:gtest",
+    ],
+)
```

Comment:
This is redundant with "testing", correct?

Reply:
Correct!